metadata-ingestion/tests/integration/clickhouse/test_clickhouse.py

```diff
@@ -39,3 +37,35 @@ def test_clickhouse_ingest(docker_compose_runner, pytestconfig, tmp_path, mock_t
             output_path=tmp_path / "clickhouse_mces.json",
             golden_path=test_resources_dir / "clickhouse_mces_golden.json",
         )
+
+
+@freeze_time(FROZEN_TIME)
+@pytest.mark.integration
```

Comment:
Thank for adding the test. This is not the test I was looking for. I was looking for unit test similar to `test_clickhouse_uri_native_secure` in test_clickhouse_source.py for clickhouse `secure` and `protocol` config that demonstrate backward compatibility if these configs are used. 

The current test looks good too, however, I believe the golden files are exactly same as existing clickhouse_mces_golden.json, except the run_id. I'd suggest using the same golden files in check_golden_file by using same run_id in new sqlalchemy uri recipe, rather than adding a duplicate 99% same golden file.